kinetic/distribution.yaml

```diff
@@ -11689,6 +11689,21 @@ repositories:
       url: https://github.com/yujinrobot/yocs_msgs.git
       version: kinetic
     status: developed
+  yoctopuce_altimeter:
+    doc:
+      type: git
+      url: https://github.com/amstrudy/yoctopuce_altimeter.git
+      version: kinetic
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/amstrudy/yoctopuce_altimeter.git
```

Comment:
This is not a release repository. It doesn't look like you have made a release. If you want to have this indexed and documented, but haven't made a release you should skip the `release` section here.